### PR TITLE
adding primary bar chart

### PR DESCRIPTION
**Pull request by ray-vazquez**

### CODE REVIEW

**Review by fnikitathomas — APPROVED** [review]
Working  code!!

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,95 @@
...
+    label: 'somethingA',
+    values: [...dG]
+  }
+
+  let dZ = {
+    label: 'somethingA',
+    values: [...dY]
+  }
+
+  console.log(dX)
```
Comment (Louis345) [inline src/App.js]: Let's get rid of this.

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,95 @@
...
+      x: parseFloat(s.current_salary.slice(1)),
+      y: parseFloat(s.previous_salary.slice(1)),
+    }
+  })
+
+  let dG = d0
+
+  dG.sort((a,b) => a.x - b.x)
+
+  console.log("dG",dG)
```
Comment (Louis345) [inline src/App.js]: This one also.

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,95 @@
...
+    return {
+      x: parseFloat(s.current_salary.slice(1)),
+      y: parseFloat(s.previous_salary.slice(1)),
+    }
+  })
+  let dF = d9
+
+  dF.sort((a,b)=> a.x - b.x)
+
+  console.log("dF",dF)
```
Comment (Louis345) [inline src/App.js]: This one as well.

File: src/App.js (hunk 2 above)
Comment (ray-vazquez) [reply src/App.js]: removed console.log but need sort function as part of what we're demonstrating.

File: src/App.js (hunk 3 above)
Comment (ray-vazquez) [reply src/App.js]: removed console.log but need sort function as part of what we're demonstrating.

File: src/App.js (hunk 1 above)
Comment (ray-vazquez) [reply src/App.js]: removed console.log but need sort function as part of what we're demonstrating.

**Review by SteveO1138 — APPROVED** [review]

**Pull request merged**